lib/src/main/java/org/team100/lib/config/NoteDetector.java

```diff
@@ -35,23 +36,42 @@ public NoteDetector(
      *         meters
      */
     public double getX() {
+
         return m_cameraAngles.getX(m_notePosition24ArrayListener.getY().get());
+
     }
 
     /**
      * @return A field relative translational x value of an object in a camera in
      *         meters
      */
     public Double fieldRelativeX() {
-            return fieldRelativePose2d().getX();
+        switch (Identity.instance) {
+            case BETA_BOT:
+            case COMP_BOT:
+                return fieldRelativePose2d().getX();
+            case BLANK:
+                return 0.0;
+            default:
+                return 0.0;
+        }
     }
 
     /**
      * @return A robot relative angle in radians to the note
      */
     public double getAngleToNote() {
-        return m_cameraAngles.getAngle(m_notePosition24ArrayListener.getX().get(),
-                m_notePosition24ArrayListener.getY().get());
+        switch (Identity.instance) {
+            case BETA_BOT:
+            case COMP_BOT:
+                return m_cameraAngles.getAngle(m_notePosition24ArrayListener.getX().get(),
+                        m_notePosition24ArrayListener.getY().get());
+            case BLANK:
+                return MathUtil.angleModulus(m_swerve.getPose().getTranslation().getAngle().getRadians() + Math.PI);
```

Comment:
what is this expression supposed to mean?  it seems like the angle to the origin (i.e. corner of the field), plus 90 degrees?